common/lib/container-definitions/src/deltas.ts

```diff
@@ -148,21 +148,6 @@ export interface IDeltaManager<T, U> extends IEventProvider<IDeltaManagerEvents>
     /** Flag to indicate whether the client can write or not. */
     readonly active: boolean;
 
-    /**
-     * Tells if container is in read-only mode.
-     * Data stores should listen for "readonly" notifications and disallow user making changes to data stores.
-     * Readonly state can be because of no storage write permission,
-     * or due to host forcing readonly mode for container.
-     *
-     * We do not differentiate here between no write access to storage vs. host disallowing changes to container -
-     * in all cases container runtime and data stores should respect readonly state and not allow local changes.
-     *
-     * It is undefined if we have not yet established websocket connection
-     * and do not know if user has write access to a file.
-     * @deprecated - use readOnlyInfo
-     */
-    readonly readonly?: boolean;
-
```

Comment:
to confirm, this was the one we talked about before and you verified has no usages in other internal repos, right?

Reply:
I just double checked, and we do actually have some references in partner repos that need to be taken care of first. Not sure why I thought this was already taken care of. We can revisit this PR after those changes are made.